features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/TenantTheme/UploadTheme.jsx

```diff
@@ -94,20 +115,47 @@ const useStyles = makeStyles((theme) => ({
 function UploadTheme() {
     const classes = useStyles();
     const [themeFile, setThemeFile] = useState([]);
+    const [fileName, setFileName] = useState('');
     const [isFileAccepted, setIsFileAccepted] = useState(false);
-    const [isUplaodUnsuccessful, setIsUploadUnsuccessful] = useState(false);
-
+    const [isUploadUnsuccessful, setIsUploadUnsuccessful] = useState(false);
+    const intl = useIntl();
+    const restApi = new API();
 
     const onDrop = (acceptedFile) => {
         setIsFileAccepted(true);
         setThemeFile(acceptedFile[0]);
     };
 
+    useEffect(() => {
+        const tenantThemeContent = restApi.exportTenantTheme();
+        tenantThemeContent.then((response) => {
+            const tenantFileName = response.headers['content-disposition'].split('filename=')[1];
+            setFileName(tenantFileName.slice(1, -1));
+        });
+    }, []);
+
+    /**
+     * Downloads Tenant Theme ZIP file.
+     *
+     */
+    const handleDownloadTenantTheme = () => {
+        const tenantThemeContent = restApi.exportTenantTheme();
+        tenantThemeContent.then((response) => {
+            Utils.forceDownload(response);
```

Comment:
We can simplify this as 

```
tenantThemeContent.then(Utils.forceDownload)
```

no need to create a lambda/arrow function there

Reply:
Fixed with `2d41afa`